hs_core/forms.py

```diff
@@ -23,460 +23,112 @@ class Helper(object):
-                                        '<h4 class="modal-title" id="myModalLabel">Delete metadata element</h4>'
-                                    '</div>'
-                                    '<div class="modal-body">'
-                                        '<strong>Are you sure you want to delete this metadata element?</strong>'
-
-                                    '</div>'
-                                    '<div class="modal-footer">'
-                                        '<button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>'),
-                                        HTML('<a type="button" class="btn btn-danger" href="">Delete</a>'),
-                                    HTML('</div>'
-                                '</div>'
-                            '</div>'
-                        '</div>'),
+                    HTML('<div class="modal fade" id="delete-metadata-element-dialog" '
+                         'tabindex="-1" role="dialog" aria-labelledby="myModalLabel" '
+                         'aria-hidden="true">'),
+                    HTML('<div class="modal-dialog">'
+                         '<div class="modal-content">'
+                         '<div class="modal-header">'
+                         '<button type="button" class="close" data-dismiss="modal" '
+                         'aria-hidden="true">&times;</button>'
+                         '<h4 class="modal-title" id="myModalLabel">'
+                         'Delete metadata element</h4>'
+                         '</div>'
+                         '<div class="modal-body">'
+                         '<strong>Are you sure you want to delete this metadata '
+                         'element?</strong>'
+
+                         '</div>'
+                         '<div class="modal-footer">'
+                         '<button type="button" class="btn btn-default" '
+                         'data-dismiss="modal">Cancel</button>'),
+                    HTML('<a type="button" class="btn btn-danger" href="">Delete</a>'),
+                    HTML('</div>'
+                         '</div>'
+                         '</div>'
+                         '</div>'),
                     )
     return layout
 
 
 class MetaDataElementDeleteForm(forms.Form):
-    def __init__(self, res_short_id, element_name, element_id , *args, **kwargs):
+    def __init__(self, res_short_id, element_name, element_id, *args, **kwargs):
         super(MetaDataElementDeleteForm, self).__init__(*args, **kwargs)
         self.helper = FormHelper()
-        self.delete_element_action = '"/hsapi/_internal/%s/%s/%s/delete-metadata/"' % (res_short_id, element_name, element_id)
+        self.delete_element_action = '"/hsapi/_internal/%s/%s/%s/delete-metadata/"' % \
+                                     (res_short_id, element_name, element_id)
```

Comment:
@pkdash better to change to the new recommended string format style by using ```format``` method.